### DISCUSSION TITLE
组件库包发布正式版本需要达到什么标准

### DISCUSSION THREAD

**Discussion by xiaosansiji** [Tencent/tdesign#121] [General]
整体通用标准
- [x] 代码测试覆盖率：覆盖率达80%+，测试用例覆盖所有交互 demo
- [ ] ~~API 稳定性：组件 API 实现情况，PMC review API 实现情况 https://github.com/Tencent/tdesign-vue/issues/1750~~
- [x] 仓库目录结构统一调整
- [x] 支持全局配置前缀 classPrefix，如 https://github.com/Tencent/tdesign-vue/issues/831
- [x] 官网走查：
    - [x] 组件 UI 调整的所有 issue 关闭 https://github.com/Tencent/tdesign/milestones （11.19）
    - [x] demo 文案、数据等调整
- [x] 新组件：https://github.com/Tencent/tdesign/issues/220
- [x] 全局样式 Token 间距相关的调整 https://github.com/Tencent/tdesign-common/issues/992 （11.18 delay 进行中）
- [ ] 正式版本 PR 稿

**Comment by xiaosansiji** [comment]
tdesign-vue-next 额外新增一条标准：composition API 重构完成

**Comment by LeeJim** [comment]
还有组件数量，web端、mobile端需要实现哪些必备组件。

**Comment by xiaosansiji** [comment]
桌面端技术栈
|     | Statements | Branches | Functions | Lines |
|  ----  | ----  |  ----  | ----  | ----  |
| Vue2  | 79.61% | 75.19% | 54.97% | 79.56% |
| Vue next  | 81.58% | 75.8% | 54.77% | 81.58% |
| React  | 60.35% | 55.01% | 53.37% | 61.58% |

**Comment by xiaosansiji** [comment]
tdesign-vue 需要适配 vue 2.7 嘛？现状是：
- tdesign-vue 支持到 vue 2.6
- tdesign-vue-next 支持到 vue 3.x

中间隔了个 2.7，可能要看下 2.7 的占有量之类的数据？
